backend/logger.js

```diff
@@ -128,12 +119,20 @@ repoLogger.prototype.logMessage = function (type, msg, meta) {
 
 	const metadata = Object.assign({}, meta);
 
-	this.session && this.session.user && (metadata.username = this.session.user.username);
-	this.req && this.req.method && (metadata.method = this.req.method);
-	this.req && this.req.originalUrl && (metadata.url = this.req.originalUrl);
+	if (this.session && this.session.user) {
+		metadata.username = this.session.user.username;
+	}
+
+	if (this.req && this.req.method) {
```

Comment:
merge the 2 ifs (kind of) so we only test this.req once
```suggestion
	if (this.req) {
	    if (this.req.method) {
	        //...
	    }
	
	}
```